frontend/src/common/MarkdownDisplay.tsx

```diff
@@ -99,6 +100,24 @@ export default function MarkdownDisplay({ children }: MarkdownDisplayProps) {
             />
           ),
         },
+        code: {
+          component: ({ children, ...props }: any) => {
+            const language = props.className ? props.className.replace('lang-', '') : ''
+
+            return (
+              <Box
+                component='code'
+                sx={{
+                  backgroundColor: theme.palette.grey[200],
+                  fontFamily: 'monospace',
+                }}
+                {...props}
+              >
+                {language ? <SyntaxHighlighter language={language}>{children}</SyntaxHighlighter> : children}
```

Comment:
Check that all colours used by this package conform to accessibility contrast requirements for both light and dark mode themes.